File: app/src/main/java/com/github/willjgriff/ethereumwallet/ui/settings/mvp/SettingsView.java
package com.github.willjgriff.ethereumwallet.ui.settings.mvp;

/**
 * Created by Will on 14/02/2017.
 */

public interface SettingsView {

	void openCreateAccountScreen();

	void showDeleteAddressDialog();

	void setActiveAddress(String address);

	void setAddressDeleted();

	void openChangeAddressScreen();
}

File: app/src/main/kotlin/com/github/willjgriff/ethereumwallet/ui/screens/settings/mvp/SettingsView.kt
package com.github.willjgriff.ethereumwallet.ui.screens.settings.mvp

/**
 * Created by williamgriffiths on 04/05/2017.
 */
interface SettingsView {

    fun openCreateAccountScreen()

    fun showDeleteAddressDialog()

    fun setActiveAddress(address: String)

    fun setAddressDeleted()

    fun openChangeAddressScreen()
}